scraper.go

```diff
@@ -146,6 +148,15 @@ func (s *Scraper) scrape() bool {
 	}
 	defer conn.Close()
 
+	if s.TLS {
+		// Reconnect with TLS
+		err = conn.StartTLS(&tls.Config{InsecureSkipVerify: true})
+		if err != nil {
+			log.Fatal(err)
```

Comment:
My two cents, but maybe you should use the same way it is done elsewhere. Something like this
`s.log.WithError(err).Error("TLS connection failed")`